docs/content/installation.md

```diff
@@ -72,17 +70,17 @@ just going to install one HVA on the same machine as dcmgr.
 
     sudo yum install -y wakame-vdc-hva-openvz-vmapp-config
 
-#### Install webui
+### Install webui
 
 This is Wakame-vdc's GUI. It's actually a [Rails application](http://rubyonrails.org) that sits in front of the dcmgr web API.
 
     sudo yum install -y wakame-vdc-webui-vmapp-config
 
-#### Reboot to load OpenVZ kernel
+### Reboot to load OpenVZ kernel
 
 These Wakame-vdc packages have installed OpenVZ as a dependency. OpenVZ runs on a custom kernel. Reboot your machine so that kernel gets loaded.
 
-#### Set up bridged networking
+### Set up bridged networking
 
```

Comment:
This does not seem right:

       83:### Set up bridged networking
      126:#### Configuration

The configuration section is not about bridged networking.